dashboard/app/models/level_source_image.rb

```diff
@@ -25,6 +25,12 @@ def save_to_s3(image)
     return false if CDO.disable_s3_image_uploads
     return false if image.blank?
 
+    begin
+      image = ImageLib::to_png(image)
```

Comment:
So ultimately this lands on S3 as a PNG as before, and hopefully the rate of sharing is not so high that our frontends can't keep up with conversion rate.  It seems unlikely with 50 frontends.  

Is there a reason why we can't do JPG everywhere?  I suppose this is lower risk.